spec/system/candidate_interface/continuous_applications/submitting/candidate_tries_to_submit_an_application_for_course_unavailable_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+RSpec.feature 'Candidate tries to submit an application choice which the course is unavailable' do
+  include SignInHelper
+  include CandidateSubmissionHelper
+  before do
+    given_i_am_signed_in
+    and_i_have_one_application_in_draft
+  end
+
+  after do
```

Comment:
I'm not a fan of using hooks to deduplicate tests by putting assertions in them.

The hooks are for setup and tear down.

The line that the test fails on will point to this block and not the test that failed. (Although the test line is available)
Maybe use a new method if duplication is a big issue.

Reply:
I can duplicate in all scenarios.

Reply:
Done